MiniApp/Classes/MiniAppLister.swift

```diff
@@ -1,2 +1,54 @@
 internal class MiniAppLister {
+    var environment: Environment
+    var miniAppClient: MiniAppClient
+
+    init(environment: Environment) {
+        self.environment = environment
+        self.miniAppClient = MiniAppClient()
+    }
+
+    func fetchList(completionHandler: @escaping (Result<[MiniAppInfo], Error>) -> Void) {
+        guard let url = environment.listingUrl else {
+            return completionHandler(.failure(self.invalidURLError()))
+        }
+
+        self.miniAppClient.requestFromServer(request: listingURLRequest(url: url)) { (result) in
+            switch result {
+            case .success(let responseData):
+                guard let decodeResponse = self.decodeListingResponse(with: responseData.data) as? [MiniAppInfo] else {
```

Comment:
I think there needs to be a separate model for the response rather than decoding directly to `MiniAppInfo`, and then you can convert the response model to `MiniAppInfo`. This is so we can avoid having the public model be dependent on the parameter names used by the API and avoid exposing things from the API response which shouldn't be exposed to the user. 

On Android, we were able to use a single class for both and just change some of the parameters to `internal`, and we can use `SerializedName` if we wish to have different parameter names than the response. Is something similar possible on iOS? 
https://github.com/rakutentech/android-miniapp/blob/master/miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/MiniAppInfo.kt

Also, the response model from the API has changed, so please check it again.